Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-slim
+LABEL maintainer="bondziks@gmail.com"
+
+ENV PYTHONNBUFFERED 1
```

Comment:
There is a typo in the environment variable name. It should be 'PYTHONUNBUFFERED' instead of 'PYTHONNBUFFERED'. This variable is used to ensure that Python output is sent straight to the terminal without being buffered.